packages/template-retail-react-app/app/components/_app/index.test.js

```diff
@@ -17,6 +17,15 @@ import messages from '@salesforce/retail-react-app/app/static/translations/compi
 import mockConfig from '@salesforce/retail-react-app/config/mocks/default'
 
 jest.mock('../../hooks/use-multi-site', () => jest.fn())
+let mockActiveDataEnable = false
+jest.mock('@salesforce/retail-react-app/app/constants', () => {
+    const mockConstants = jest.requireActual('@salesforce/retail-react-app/app/constants')
+    return {
+        __esModule: true,
+        ...mockConstants,
+        ACTIVE_DATA_ENABLE: mockActiveDataEnable
+    }
+})
```

Comment:
This mock setup doesn't have the intended behavior. The `jest.mock` setup is only done once; modifying `mockActiveDataEnable` after that does not change the `ACTIVE_DATA_ENABLE` constant.

Instead, you should use a jest spy or mock, and reset it in a `beforeEach` to ensure it's a clean value every time.

Reply:
removed